orbit/pkg/packaging/windows_templates.go

```diff
@@ -102,3 +102,62 @@ var windowsWixTemplate = template.Must(template.New("").Option("missingkey=error
   </Product>
 </Wix>
 `))
+
+var windowsOsqueryEventLogTemplate = template.Must(template.New("").Option("missingkey=error").Parse(
+	`<?xml version="1.0"?>
+<instrumentationManifest xsi:schemaLocation="http://schemas.microsoft.com/win/2004/08/events eventman.xsd" xmlns="http://schemas.microsoft.com/win/2004/08/events" xmlns:win="http://manifests.microsoft.com/win/2004/08/windows/events" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:trace="http://schemas.microsoft.com/win/2004/08/events/trace">
+	<instrumentation>
+		<events>
+			<provider name="FleetDM" guid="{F7740E18-3259-434F-9759-976319968900}" symbol="OsqueryWindowsEventLogProvider" resourceFileName="%systemdrive%\Program Files\Orbit\bin\osqueryd\windows\{{ .OsquerydChannel }}\osqueryd.exe" messageFileName="%systemdrive%\Program Files\Orbit\bin\osqueryd\windows\{{ .OsquerydChannel }}\osqueryd.exe">
```

Comment:
sanity check: I couldn't find anything relevant [here](https://learn.microsoft.com/en-us/windows/win32/wes/identifying-the-provider), but do you know if we should make this guid unique to us? I see this is using the same value as [this template](https://raw.githubusercontent.com/osquery/osquery/master/tools/wel/osquery.man).

Reply:
Good question! This GUID is unique to osquery and has to match the GUID defined by the osquery's eventlog logger plugin  [here](https://github.com/osquery/osquery/blob/master/plugins/logger/generated_wel.h#L11). If GUID does not match,  the eventlog registration through this manifest will not work.